README.md

```diff
@@ -103,6 +103,31 @@ import { EntityOpsLevelMaturityContent } from '@opslevel/backstage-maturity';
     </EntityLayout.Route>
 ```
 
-## Next Steps
+### Next Steps
 
-Visit the Maturity tab in Backstage to get started.
+Visit the Maturity tab in Backstage to get started. Once you have synced data from Backstage to OpsLevel, you will likely want to set up auto-syncing of data using the [backend-plugin](https://github.com/OpsLevel/backstage-plugin-backend).
+
+
+## Troubleshooting
+
+### 403 API Requests
+
+Please validate that you do not have any middleware set up in Backstage that could be removing cookies or headers from the request by the Backstage proxy.
+
+### 404 API Requests
+
+If you are receiving 404s, it is possible that there is a network firewall preventing Backstage from accessing the OpsLevel instance.
+
+You can validate this by using [Postman](https://www.postman.com/downloads/) or another tool to simulate the request from the machine running Backstage.
+
+Sometimes it can be hard to set up a tool in environments running Backstage, in which was you can retrieve headers and cookies from a browser logged into OpsLevel and simulate the request with `curl`.
```

Comment:
I think we'd make it a lot easier for the user, if we just suggested to set the Authorization: Bearer <api_token>

Reply:
Updated!